Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: alpaca
+alpaca:
+	go build
+
+.PHONY: lint
+lint:
+	golangci-lint run
+
+.PHONY: coverage.txt
+coverage.txt:
+	go-acc ./...
+
+coverage.html: coverage.txt
+	go tool cover -html=coverage.txt -o coverage.html
+
+.PHONY: test
+test: coverage.html lint
+	xdg-open coverage.html
```

Comment:
`go tool cover` has logic that works on different OSes to open the coverage output in the browser.

I don't know what `go-acc` is that generates coverage.txt, but what we usually do is:
```
test:
    go test -coverprofile=out/coverage.txt

cover: test
    go tool cover -html=out/coverage.txt
```
See https://github.com/foxygoat/s/blob/92c4b00e14a3576e9dc4ca99b1ca94e0505188f6/Makefile#L25 for an example.

Reply:
`go-acc` is another coverage tool that also collects coverage for integration tests (that run across packages). I like having that visibility there, but I guess it is not a standard part of the go toolchain, and I'm not sure if it has a way to open a browser in a cross-platform way.

I'm a bit hesitant to introduce strict test coverage requirements into Alpaca as you've done in the foxygoat/s repo. So I'm going to remove the Makefile from this PR and we can figure it out later.